### ISSUE TITLE
Unable to compile app after upgrading to 1.7.2

### ISSUE THREAD

**Issue by RnbWd** [react-native-push-notification-ios/push-notification-ios#226]
I am getting the following error after running `npx react-native run-ios --simulator 'iPhone 12'`

```
The following build commands failed:
	CompileC /Users/dev/Library/Developer/Xcode/DerivedData/BrightID-eratlntujddnpaewlhvxbkufgvjj/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/RNCPushNotificationIOS.build/Objects-normal/x86_64/RNCPushNotificationIOS.o /Users/dev/BrightID/BrightID/BrightID/node_modules/@react-native-community/push-notification-ios/ios/RNCPushNotificationIOS.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```

unable to compile and build my app with v1.7.2

**Comment by thejogsa** [comment]
I was also getting same issue in `1.7.2`, Now I downgraded on` 1.7.1 version` and Now working fine.

**Comment by aminhdev** [comment]
> I am getting the following error after running `npx react-native run-ios --simulator 'iPhone 12'`
> 
> ```
> The following build commands failed:
> 	CompileC /Users/dev/Library/Developer/Xcode/DerivedData/BrightID-eratlntujddnpaewlhvxbkufgvjj/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/RNCPushNotificationIOS.build/Objects-normal/x86_64/RNCPushNotificationIOS.o /Users/dev/BrightID/BrightID/BrightID/node_modules/@react-native-community/push-notification-ios/ios/RNCPushNotificationIOS.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
> (1 failure)
> ```
> 
> unable to compile and build my app with v1.7.2

Same issue

**Comment by JoaoPauloCMarra** [comment]
https://github.com/react-native-push-notification-ios/push-notification-ios/commit/ba6bc11f862ac79f78c8442b9d2965cc942aef26#comments

same for me

**Comment by raymatos** [comment]
Can you remove the version, seems like this is an inner dep for other projects and its causing issues

**Comment by Naturalclar** [comment]
Sorry for all the trouble caused. I've published v1.7.3 that will fix the issue.
